app/src/main/java/org/mozilla/fenix/search/SearchFragmentStore.kt

```diff
@@ -213,7 +213,7 @@ sealed class SearchFragmentAction : Action {
     /**
      * Updates the local `SearchFragmentState` from the global `SearchState` in `BrowserStore`.
      */
-    data class UpdateSearchState(val search: SearchState) : SearchFragmentAction()
+    data class UpdateSearchState(val search: SearchState, val settings: Settings) : SearchFragmentAction()
```

Comment:
Instead of passing the entire `Settings`, could we just pass in the necessary Boolean value that is needed?